sdks/solana/stork-sdk/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "stork-sdk"
+version = "0.1.0"
+edition = "2021"
+
+
+[lib]
+crate-type = ["lib"]
+name = "stork_sdk"
+
+[dependencies]
+anchor-lang = "0.30.1"
+solana-program = ">=1.16.0"
```

Comment:
solana-program dependency doesn't seem to be used in the sdk anywhere and can likely be removed.